### PR TITLE
Release post for Node.js 14.15.0

### PR DESCRIPTION
**Pull request by richardlau**
Refs: https://github.com/nodejs/node/pull/35746

### CODE REVIEW

**Review by ruyadorno — APPROVED** [review]

**Comment by Sid-ah** [comment]
We need to remove the <a> tag as Twitter doesn't support it (I was wrong).

**Review by BethGriggs — APPROVED** [review]

**Comment by richardlau** [comment]
I opened this as I don't have a Twitter account myself and thought it'd be fun to try out this Twitter-together thing. @bnb has retweeted @cjihrig's [tweet](https://twitter.com/cjihrig/status/1321125637248159749) so I'll leave it up to @nodejs/social as to whether to close this PR ¯\\_(ツ)\_/¯.

**Comment by bnb** [comment]
@richardlau do you mind if we wait a day and tweak it a bit (I can use GitHub's suggestions feature) to follow up with the retweet as a reminder/FYI?

**Comment by richardlau** [comment]
> 
> 
> @richardlau do you mind if we wait a day and tweak it a bit (I can use GitHub's suggestions feature) to follow up with the retweet as a reminder/FYI tomorrow?

Don't mind at all 😀.

File: tweets/releases/v14.15.0.tweet [hunk 1]
```diff
@@ -0,0 +1 @@
+Node.js 14.15.0 'Fermium' (LTS) has now been released. This is the first LTS version of Node.js 14. https://nodejs.org/en/blog/release/v14.15.0/
```
Comment (bnb) [inline tweets/releases/v14.15.0.tweet:1]:
```suggestion
🎉  Node.js 14.15.0 'Fermium' (LTS) has now been released. This is the first LTS version of Node.js 14. 🎉 
https://nodejs.org/en/blog/release/v14.15.0/
```

**Pull request merged**